plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -102,6 +103,18 @@ export function WriteItForMeButtonContainer() {
 			},
 		} );
 
+	const { requestCompletion: requestShortCompletion } = useCompletion( {
+		feature: WOO_AI_PLUGIN_FEATURE_NAME,
+		onStreamMessage: ( content ) => shortTinyEditor.setContent( content ),
+		onStreamError: ( error ) =>
+			shortTinyEditor.setContent( getApiError( error ) ),
+		onCompletionFinished: ( reason, content ) => {
+			if ( reason === 'finished' ) {
+				shortTinyEditor.setContent( content );
```

Comment:
We might want to add an indicator or a message on the UI that shows we are generating the short description too, and when it's finished it should indicate that we've modified the short description. Because it all happens automatically, the user might not be aware of the new changes.

Additionally, I think the feedback box should now appear after both descriptions are generated.

Reply:
> We might want to add an indicator or a message on the UI that shows we are generating the short description too, and when it's finished it should indicate that we've modified the short description. Because it all happens automatically, the user might not be aware of the new changes.

I don't disagree, although again I'm uncertain how this would show up. I'm not sure it's worth complicating the UI or code all that much, as the short description is really just a "bonus" summary of the long description. As mentioned in my other comment I think moving to a modal would clarify this kind of thing.

> Additionally, I think the feedback box should now appear after both descriptions are generated.

I thought about this but I decided to just leave it as it for now, although it's definitely debatable. This is what I was thinking:
- I think the feedback should continue to just reflect the outcome of the primary description as the short description is just a summary, and this would continue to align with how the tracks event has been capturing this up to this point.
- Often times the short description isn't actually visible in the user's viewport, so this will just look like an awkward pause where nothing is happening if we delay the feedback. 
- I still think of the short description as something of a less-critical "bonus," and doesn't need to play much of a part in this immediate interaction. They'll likely  just notice it when continuing down the page as they create their product.

Reply:
I don't know how feasible it is, but it'd be nice to show a small button fixed at the bottom of the screen once the summary is generated. Clicking it would scroll down to the short description field, letting the user know there's new content they have yet to see. It could look like this:

<img width="330" alt="image" src="https://github.com/woocommerce/woocommerce/assets/79307566/d3eccf21-2360-452f-aa5e-2030afb30433">

We could then show a spotlight message to inform them how the description was generated (it'd only show up once, so the user isn't bugged with it every time they add a product). It could show up when the short description field comes into view.

<img width="818" alt="image" src="https://github.com/woocommerce/woocommerce/assets/79307566/8a741641-f433-46dd-a21e-febf05f5e148">

A low-effort version of this would show a snackbar mentioning the two descriptions. It'd only be shown after both descriptions are generated.

<img width="417" alt="image" src="https://github.com/woocommerce/woocommerce/assets/79307566/b1de8b83-f609-43d3-9ffa-4a1f17e8f15a">

As for modifying existing descriptions, I'd be wary of that. Can we not generate the short description if there's already one? In that case, it makes sense to consider it while generating the long description. I'm mentioning this because many merchants use the short description _instead_ of the long description because that's how their themes are structured.

> I still think of the short description as something of a less-critical "bonus," and doesn't need to play much of a part in this immediate interaction. They'll likely just notice it when continuing down the page as they create their product.

Can you show me what this looks like currently?

Reply:
Thanks @jarekmorawski , I really like it! As it stands we've just been overwriting any description when the user requests another one via the "Write with AI" button, as well as printing errors to the editor if they occur. This is what it looks like:
![image](https://github.com/woocommerce/woocommerce/assets/444632/5067e01d-40f4-44af-af1d-d592abf542ff)


Although I've actually just made a change to display errors via the snackbar instead, at least temporarily until we can implement something like the above.